lib/parsers/cr3_parser/decoder.rb

```diff
@@ -0,0 +1,34 @@
+require 'parsers/iso_base_file_format/decoder'
+
+class FormatParser::CR3Parser::Decoder < FormatParser::Decoder
```

Comment:
Tests?

Reply:
The tests in the cr3_parser_spec cover the functionality here.